### PR TITLE
style form errors

### PR DESCRIPTION
**Pull request by localgrr**

### CODE REVIEW

File: euth/user_management/templates/user_management/register.html [hunk 1]
```diff
@@ -25,30 +26,46 @@
...
+                {{ form.password_repeat|add_class:"form-control form-error" }}
+            {% else %}
+                {{ form.password_repeat|add_class:"form-control"}}
+            {% endif %}
             {{ form.password_repeat.errors }}
-            {{ form.password_repeat|add_class:"form-control" }}
         </div>
 
-        <button type="submit" value="register" class="btn btn-primary btn-cta">{% trans "register" %}</button>
+        <button type="submit" value="Register" class="btn btn-primary btn-cta">Register</button>
```
Comment (slomo) [inline euth/user_management/templates/user_management/register.html]: Why did you remove the translation tag for "Register"?

File: euth/user_management/templates/user_management/login.html [hunk 2]
```diff
@@ -27,22 +29,30 @@
...
+                {% if form.password.errors %}
+                    {{ form.password|add_class:"form-control form-error" }}
+                {% else %}
+                    {{ form.password|add_class:"form-control"}}
+                {% endif %}
+                {{ form.password.errors }}
             </div>
 
-            <button type="submit" value="Login" class="btn btn-primary btn-cta">{% trans "login" %}</button>
+            <button type="submit" value="Login" class="btn btn-primary btn-cta">Anmelden</button>
```
Comment (slomo) [inline euth/user_management/templates/user_management/login.html]: Also here translation tag has been removed.

File: euth/user_management/templates/user_management/login.html [hunk 3]
```diff
@@ -3,6 +3,8 @@
 
 {% block content %}
 <div class="container">
+<h2>{% trans "Login" %}</h2>
+
```
Comment (slomo) [inline euth/user_management/templates/user_management/login.html]: Nico asked me to remove those.

**Comment by slomo** [comment]
Can you maybe merge or rebase master?

File: euth/user_management/templates/user_management/login.html (hunk 2 above)
Comment (localgrr) [inline euth/user_management/templates/user_management/login.html]: idk it was an accident
